build/make-manifests.sh

```diff
@@ -53,13 +43,7 @@ for MF in deploy/service_account.yaml deploy/role.yaml deploy/role_binding.yaml
 done
 ) > ${CLUSTER_VERSIONED_DIR}/kubevirt-ssp-operator.yaml
 
-# TODO: we should use --from-version
-# note: this creates under deploy/olm-catalog ...
-${OPERATOR_SDK} olm-catalog gen-csv --csv-version ${VERSION}
-
-./build/update-olm.py \
-	deploy/olm-catalog/kubevirt-ssp-operator/${VERSION}/kubevirt-ssp-operator.${TAG}.clusterserviceversion.yaml > \
-	${MANIFESTS_VERSIONED_DIR}/kubevirt-ssp-operator.${TAG}.clusterserviceversion.yaml
+./build/csv-generator.sh --csv-version=${VERSION} --namespace=placeholder --operator-image=REPLACE_IMAGE > ${MANIFESTS_VERSIONED_DIR}/kubevirt-ssp-operator.${TAG}.clusterserviceversion.yaml
```

Comment:
Just to see that were not missing something,  if we run make-manifests ,e.g. when releasing a tag - triggered by .travis.yaml : `make release "IMAGE_TAG=$TRAVIS_TAG"` 
where is the swap/input for `REPLACE_IMAGE` coming from ?

Reply:
> where is the swap/input for REPLACE_IMAGE coming from ?

The release CSVs are still using the "REPLACE_IMAGE" image value. I don't know if that's intentional or not. I'm just replicating the existing behavior 1 for 1. 

For example, take a look at the v1.0.5 CSV (https://github.com/MarSik/kubevirt-ssp-operator/releases/download/v1.0.5/kubevirt-ssp-operator.v1.0.5.clusterserviceversion.yaml) It still has REPLACE_IMAGE in the released CSV. 

I noticed other odd things as well in the release process. For example the release kubevirt-ssp-operator.yaml for v1.0.5 (https://github.com/MarSik/kubevirt-ssp-operator/releases/download/v1.0.5/kubevirt-ssp-operator.yaml) is using the floating "latest" tag, which basically means there's no reason in making releases because they all point to the "latest" release container image regardless of the version of the manifest that is installed.

I didn't want to get involved in all that, so I just preserved whatever behavior currently exists for the ssp release process. For the purposes of the HCO, the csv-generator script lets us have the control needed to build the CSV accurately in the HCO's CSV build process regardless of what the ssp's release process is doing.

Reply:
Where do you see latest? REPLACE_IMAGE can easily be replaced with the hash or proper name:tag pair.

Reply:
> Where do you see latest

in the kubevirt-ssp-operator.yaml manifest for each release. 

```
wget https://github.com/MarSik/kubevirt-ssp-operator/releases/download/v1.0.5/kubevirt-ssp-operator.yaml && cat kubevirt-ssp-operator.yaml | grep latest
.
.
.
          image: quay.io/fromani/kubevirt-ssp-operator-container:latest
```

Reply:
Ah, that is the manifest file with filled in data. This looks like a bug. @fromanirh

Reply:
Indeed REPLACE_IMAGE should be replaced (no pun intended) by our automation when we do release. @davidvossel not sure we should fix before or after your PR, whatever works best for you

Reply:
For reference:
operator image should be
1. on travis, since we are going to test the image out of the sources BEFORE the upload on quay.io:
`quay.io/fromani/kubevirt-ssp-operator-container:$TAG`, where `tag` could be in turn `devel` or `$TRAVIS_TAG`, aka the current tag
2. everywhere else, the only thing which changes is the tag, which should be the last released tag, found using github APIs or any other means.